astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,354 @@
+        # thankfully, adding extra planet-specific keywords here does not break query for other planets
+        request_payload = dict(
+            [
+                ("abbrev", planet[:3]),
+                ("ephem", self.planet_defaults[planet]["ephem"]),
+                ("time", epoch.utc.iso[:16]),
+                ("fov", 10),  # next few are figure options, can be hardcoded and ignored
+                ("fov_unit", planet.capitalize() + " radii"),
+                ("center", "body"),
+                ("center_body", planet.capitalize()),
+                ("center_ansa", self.planet_defaults[planet]["center_ansa"]),
+                ("center_ew", "east"),
+                ("center_ra", ""),
+                ("center_ra_type", "hours"),
+                ("center_dec", ""),
+                ("center_star", ""),
+                ("viewpoint", viewpoint),
+                ("observatory", "Earth's center"),  # has no effect if viewpoint != observatory so can hardcode. no plans to implement calling observatories by name since ring node only names like 8 observatories
+                ("latitude", latitude),
+                ("longitude", longitude),
+                ("lon_dir", "east"),
+                ("altitude", altitude),
+                ("moons", self.planet_defaults[planet]["moons"]),
+                ("rings", self.planet_defaults[planet]["rings"]),
+                ("arcmodel", conf.neptune_arcmodels[int(neptune_arcmodel)]),
+                ("extra_ra", ""),  # figure options below this line, can all be hardcoded and ignored
+                ("extra_ra_type", "hours"),
+                ("extra_dec", ""),
+                ("extra_name", ""),
+                ("title", ""),
+                ("labels", "Small (6 points)"),
+                ("moonpts", "0"),
+                ("blank", "No"),
+                ("opacity", "Transparent"),
+                ("peris", "None"),
+                ("peripts", "4"),
+                ("arcpts", "4"),
+                ("meridians", "Yes"),
+                ("output", "html"),
+            ]
+        )
+
+        # return request_payload if desired
+        if get_query_payload:
+            return request_payload
+
+        # query and parse
+        response = self._request(
+            "GET", self.url, params=request_payload, timeout=self.timeout, cache=cache
+        )
+
+        return response
+
+    def _parse_result(self, response, verbose=None):
+        """
+        Routine for parsing data from ring node
+
+        Parameters
+        ----------
+        self : RingNodeClass instance
+        response : list
+            raw response from server
+
+
+        Returns
+        -------
+        bodytable : `astropy.QTable`
+        ringtable : `astropy.QTable`
+        """
+        self.last_response = response
+        try:
+            self._last_query.remove_cache_file(self.cache_location)
+        except (FileNotFoundError, AttributeError):
```

Comment:
It looks like this `AttributeError` case is hiding some actual problem that should be fixed instead.